chimney-cats/src/main/scala/io/scalaland/chimney/cats/validated/package.scala

```diff
@@ -0,0 +1,33 @@
+package io.scalaland.chimney.cats
+
+import io.scalaland.chimney.{TransformationContext, TransformationException}
+
+package object validated {
+  type NEC[+A] = cats.data.NonEmptyChain[A]
+  val NEC = cats.data.NonEmptyChain
+
+  type V[+A] = cats.data.Validated[NEC[TransformationError], A]
+  val V = cats.data.Validated
+
+  implicit val vContextError: TransformationContext[V] = new TransformationContext[V] {
```

Comment:
Do you think providing instance for any applicative functor would be possible?

Reply:
I think it's possible to provide instance for any applicative error functor, but it can be fast-fail and it's not nice for such transformations